doc/source/user_guide/io.rst

```diff
@@ -2831,8 +2831,28 @@ See the :ref:`cookbook<cookbook.excel>` for some advanced strategies.
 
 .. warning::
 
-  Please do not report issues when using ``xlrd`` to read ``.xlsx`` files.
-  This is not supported, switch to using ``openpyxl`` instead.
+   The `xlwt <https://xlwt.readthedocs.io/en/latest/>`_ package for writing old-style ``.xls``
+   excel files is no longer maintained.
+   The `xlrd <https://xlrd.readthedocs.io/en/latest/>`_ package is now only for reading
+   old-style ``.xls`` files.
+
+   Previously, the default argument ``engine=None`` to :func:`~pandas.read_excel`
+   would result in using the ``xlrd`` engine in many cases, also for new
+   Excel 2007+ (``.xlsx``) files.
+   If `openpyxl <https://openpyxl.readthedocs.io/en/stable/>`_  is installed,
+   many of these cases will now default to using the ``openpyxl`` engine.
+   See the :func:`read_excel` documentation for more details.
+
+   Thus, it is strongly encouraged to install ``openpyxl`` to read Excel 2007+
+   (``.xlsx``) files.
+   Please do not report issues when using ``xlrd`` to read ``.xlsx`` files.
+   This is no longer supported, switch to using ``openpyxl`` instead.
```

Comment:
I would prefer these two lines to be further up, around 2938, and in their own paragraph. I mean, I'd prefer them to be bright red and bold, maybe flashing too, but I doubt you'd agree to that ;-)